plugins/cache/redis.py

```diff
@@ -18,6 +18,7 @@
           - A colon separated string of connection information for Redis.
           - The format is C(host:port:db:password), for example C(localhost:6379:0:changeme).
           - To use encryption in transit, prefix the connection with C(tls://), as in C(tls://localhost:6379:0:changeme).
+          - To use redis sentinel, use sepparator C(;), for example C(localhost:26379;localhost:26379;0:changeme). Requires redis>=2.9.0
```

Comment:
```suggestion
          - To use redis sentinel, use sepparator C(;), for example C(localhost:26379;localhost:26379;0:changeme). Requires redis>=2.9.0.
```

Reply:
While we're at it: separator is written with one p, not with two.